lib/lambda_ethereum_consensus/state_transition/misc.ex

```diff
@@ -58,4 +108,17 @@ defmodule LambdaEthereumConsensus.StateTransition.Misc do
     slots_per_epoch = ChainSpec.get("SLOTS_PER_EPOCH")
     epoch * slots_per_epoch
   end
+
+  @spec bytes_to_uint64(binary()) :: SszTypes.uint64()
+  defp bytes_to_uint64(value) do
+    # Converts a binary value to a 64-bit unsigned integer
+    <<first_8_bytes::binary-size(8), _::binary>> = value
+    first_8_bytes |> :binary.decode_unsigned(:little)
```

Comment:
You can decode binaries as numbers as part of a match (and it can be moved to the function signature, too).
```suggestion
    <<first_8_bytes::unsigned-integer-little-size(64), _::binary>> = value
    first_8_bytes
```